src/EditorFeatures/Test2/FindReferences/FindReferencesTests.Tuples.vb

```diff
@@ -0,0 +1,234 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
+    Partial Public Class FindReferencesTests
+        Dim tuple2Doc As XElement =
```

Comment:
@CyrusNajmabadi Didn't we add some support in core TestWorkspace to introduce this type?